### ISSUE TITLE
Team 3: Which airline had the most instances of damage for a given number of strikes?  Does this airline have any indications that it is less safe than others?

### ISSUE THREAD

**Issue by alne4294** [CSCI-4830-002-2014/hackathon-birdstrikes#17]

**Comment by peymanmortazavi** [comment]
![b](https://cloud.githubusercontent.com/assets/5572218/4451695/1ac89a88-483a-11e4-94e2-8772f33b5e13.png)

Our data does not provide enough information about airlines since the airline with the most instances is 'Unknown'! The second is Military !

**Comment by chrisbopp** [comment]
Taking into account the number of in-flight hours, what is the normalized rate? Military planes might just be in the air more.

**Comment by peymanmortazavi** [comment]
Good point, however, we cannot calculate the ratio of the flights/incidents since our data does not provide the total number of flights.
